Add an alternative to DeleteVersions that leaves no garbage (#324)
cosmos/iavl: CHANGELOG.md

# Changelog

## Unreleased

### Bug Fixes

> - [\#324](https://github.com/cosmos/iavl/pull/324) Fix `DeleteVersions` causing a memory leak. 
>   Add `DeleteVersionsRange` to delete in range (@klim0v)

Reviewer: ```suggestion
- [\#324](https://github.com/cosmos/iavl/pull/324) Fix `DeleteVersions` not properly removing
  orphans, and add `DeleteVersionsRange` to delete a range (@klim0v)
```
